outbound-dialer/src/main/java/org/motechproject/nms/outbounddialer/service/impl/TargetFileServiceImpl.java

```diff
@@ -165,27 +168,203 @@ private File createTargetFileDirectory() {
     }
 
 
-    private void writeSubscriptionRow(Subscription subscription, OutputStreamWriter writer) throws IOException {
-        //subscription id
-        writer.write(subscription.getSubscriptionId());
+    private void writeSubscriptionRow(String fileIdentifier, Subscription subscription, String imiServiceId,
```

Comment:
Per #180, we need to add one more column to the file that tells IMI how the user was subscribed (IVR or MCTS_UPLOAD), which you can get from the Subscription.